src/app/cluster/cluster-details/constraints/component.ts

```diff
@@ -124,6 +131,18 @@ export class ConstraintsComponent implements OnInit, OnChanges, OnDestroy {
     return violations ? violations.length : 0;
   }
 
+  filter(): void {
+    this.dataSource.data = this.constraints.filter(constraint => {
+      let isVisible = true;
+
+      if (this.constraintTemplateFilter) {
+        isVisible = isVisible && constraint.spec.constraintType === this.constraintTemplateFilter;
+      }
+
+      return isVisible;
```

Comment:
`return this.constraintTemplateFilter ? constraint.spec.constraintType === this.constraintTemplateFilter : true` does the same thing.

Reply:
Yes. I wasn't sure if we might want to add some more filters later 🤔 But yeah, maybe lets start simple.

Reply:
You can skip the return because it will be one-liner in that case.

Reply:
Let's always keep it simple until there is no need to add anything else. Please also remember about my comment above :)